Jenkinsfile_CNP

```diff
@@ -8,10 +8,26 @@ def component = "bulk-scan-processor"
 
 def channel = '#rpe-build-notices'
 
+List<LinkedHashMap<String, Object>> secrets = [
+  secret('test-scan-delay', 'TEST_SCAN_DELAY'),
```

Comment:
Do we really need these secrets as  you are using SAS token to upload to blob storage.

Reply:
True, thanks for spotting that.